pkg/ddevapp/drupal.go

```diff
@@ -166,8 +166,8 @@ $settings['trusted_host_patterns'] = ['.*'];
 $settings['class_loader_auto_detect'] = FALSE;
 
 // This specifies the default configuration sync directory.
-if (empty($config_directories[CONFIG_SYNC_DIRECTORY])) {
-  $config_directories[CONFIG_SYNC_DIRECTORY] = '{{ joinPath $config.SitePath $config.SyncDir }}';
+if (empty($settings['config_sync_directory'])) {
+  $settings['config_sync_directory'] = '{{ joinPath $config.SitePath $config.SyncDir }}';
```

Comment:
Let's keep both of these here. It should work on most any version of Drupal 8 then true?

Reply:
Right, this should working.